### ISSUE TITLE
Alibaba R-BERT Implementation

### ISSUE THREAD

**Issue by phucdev** [phucdev/TL_Bio_RE#1]
Net initialization:

- [x] Set up fully connected layers for entities and CLS.

- [x] Set up fully connected layer for concatenation (`torch.cat((cls, e1, e2), 1)`), softmax layer.

Forward:

- [x] Retrieve hidden states for entities and perform averaging. Check if positional markers are unambiguous enough, i.e. not used normally in text. Maybe use double `$` and `#` instead.
- [x] Use same net (fully connected layer + activation) on averaged vector representations of the two entities as input, but save output for entities separately. (Parameter sharing)
- [x] Fully connected layer + activation on CLS representation.
- [x] Feed concatenation of hidden states of entities + CLS to fully connected layer, softmax.

Remarks:
We still need to retrieve the relevant entity embeddings to feed them to the subsequent layers.
Possible solutions:

1. Search for positional markers during the forward method in model code.
2. Add positional information to data files (tsv, jsonl) and use in data set reader and model code.
